puppies/settings.py

```diff
@@ -120,3 +120,6 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+LOGIN_REDIRECT_URL = '/loggedin/post/'
```

Comment:
[minor]
These should not be hard-coded, but instead use `django.urls.reverse_lazy()` to look up the url by name.

Otherwise, this is a really good way to do this!